tests/docker_images/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Dockerfile that installs imap-data-access and its dependencies
+FROM public.ecr.aws/docker/library/python:3.10-slim
+USER root
+
+# Upgrade pip to the latest version
+RUN pip install --upgrade pip
+
+RUN pip install requests
+
+# TODO: delete this section once imap-data-access is released
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y git
+RUN pip install git+https://github.com/IMAP-Science-Operations-Center/imap-data-access.git
+
+# Create the /mnt/spice directory
+WORKDIR /data
```

Comment:
# I think that you might be able to set the `IMAP_DATA_DIR` environment variable here to make sure you are in `/data`, otherwise my guess is the file might actually be in `/data/data/imap` with the double nested "data".
```suggestion
WORKDIR /data
ENV IMAP_DATA_DIR /data
```

Reply:
I didn’t notice double data folder. Will check it.